charts/kyverno-operator/README.md

```diff
@@ -6,50 +6,76 @@
 
 This chart bootstraps a Kyverno Operator on a [Kubernetes](http://kubernetes.io) cluster with [Kyverno](https://kyverno.io) installed, using the [Helm](https://helm.sh) package manager.
 
-## Installation
+## Prerequisite
+### Get token from Nirmata support
+You need token to download the images. Please contact <support@nirmata.com> to get it.
 
-```
-# 1. Add Kyverno Helm Repository
+## Installation
 
+### 1. Add Kyverno Helm Repository
+```console
 helm repo add nirmata https://nirmata.github.io/kyverno-charts/
+helm repo update nirmata
+```
 
-# 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+### 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+Create the namespace
+```console
+kubectl create namespace nirmata-kyverno-operator
```

Comment:
We can remove this step and add `--create-namespace` in the following command

Reply:
kubectl does not have a --create-namespace option. Does it?

Reply:
@atulatnirmata I meant to put this in `helm install ` command

Reply:
The namespace is created for the configmap, which has to be done before the helm install.
The helm install already has `--create-namespace`

Reply:
Ohh, Okay. No need to merge.